src/Text/Pandoc/Writers.hs

```diff
@@ -148,10 +149,10 @@ writers = [
   ,("docbook"      , TextWriter writeDocbook5)
   ,("docbook4"     , TextWriter writeDocbook4)
   ,("docbook5"     , TextWriter writeDocbook5)
-  ,("jats"         , TextWriter writeJATSarchiving)
-  ,("jats_articleauthoring", TextWriter writeJATSarticleauthoring)
-  ,("jats_publishing" , TextWriter writeJATSpublishing)
-  ,("jats_archiving" , TextWriter writeJATSarchiving)
+  ,("jats"         , TextWriter writeJATS)
```

Comment:
I'd just use writeJatsAarchiving here, so it's clear and explicit.
(And we may want to deprecate writeJATS at some point.)